### ISSUE TITLE
live-dashboard: Distinguish between invited and registered volunteers

### ISSUE THREAD

**Issue by dannypernik** [opencurrents/opencurrents#420]
Currently, we are not really showing a difference to end user, but it's okay for now. @nickolashe are we able to list these separately right now?

**Comment by nickolashe** [comment]
i don't think we are distinguishing between the two on the backend at this time
